#### Peer-reviewed Journal Articles:

1. **Paudel D**, Dareus R, Rosenwald J, Munoz-Amatriain M, Rios E. 2021. Genome-wide association study reveals candidate genes for flowering time in cowpea (*Vigna unguiculata* [L.] Walp). [Frontiers in Genetics](https://www.frontiersin.org/articles/10.3389/fgene.2021.667038/full).
2.	Dareus R, Acharya J, **Paudel D**, Gouveia BT, Lopes de Souza C, Chase C, DiGennaro P, Rios EF. 2021. Phenotypic characterization of the UC-Riverside cowpea mini-core collection for phenological and agronomic traits in Florida. [Crop Science](https://acsess.onlinelibrary.wiley.com/doi/abs/10.1002/csc2.20544).
3.	**Paudel DR**, Liu F, Wang L, Crook M, Maya S*, Peng Z, Kelley K, Ane JM, Wang J. 2020. Isolation, characterization, and complete genome sequence of a Bradyrhizobium strain Lb8 from nodules of peanut utilizing crack entry infection. [Frontiers in Microbiology](https://www.frontiersin.org/articles/10.3389/fmicb.2020.00093).
4.	Peng Z, Zhao Z, Clevenger JP, **Paudel D**, Ozias-Akins P, Wang J. 2020. Comparison of SNP calling pipelines and NGS platforms to predict the genomic regions harboring candidate genes for nodulation in cultivated peanut. [Frontiers in Genetics](https://www.frontiersin.org/articles/10.3389/fgene.2020.00222/full).
5.	Yang X, Song J, Todd J, Peng Z, **Paudel DR**, Luo Z, Ma X, You Q, Hanson E, Zhao Z, Zhao Y, Zhang J, Ming R, Wang J. 2019. Target enrichment sequencing of 307 germplasm accessions identified ancestry of ancient and modern hybrids and signatures of adaptation and selection in sugarcane (*Saccharum* spp.), a “sweet” crop with “bitter” genomes. [Plant Biotechnology Journal](https://onlinelibrary.wiley.com/doi/full/10.1111/pbi.12992): 1–11.
6.	**Paudel DR**, Kannan B, Yang X, Harris-Shultz KR, Thudi M, Varshney RK, Altpeter F, Wang J. 2018. Surveying the genome and constructing a high-density genetic map of napiergrass (*Cenchrus purpureus* Schumach). [Scientific Reports](https://www.nature.com/articles/s41598-018-32674-x) 8: 1–11.
7.	Varshney RK, Shi C, … [18 authors] … **Paudel DR**, … [42 authors] …, Vigouroux Y, Xu X. 2017. Pearl millet genome sequence provides a resource to improve agronomic traits in arid environments. [Nature Biotechnology](https://www.nature.com/articles/nbt.3943) 35: 969–976.
8.	Yang X, Song J, You Q, **Paudel DR**, Zhang J, Wang J. 2017. Mining sequence variations in representative polyploid sugarcane germplasm accessions. [BMC Genomics](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3980-3) 18: 1–16.
9.	Tian W, **Paudel DR**, Vendrame W, Wang J. 2017. Enriching genomic resources and marker development from transcript sequences of *Jatropha curcas* for microgravity studies. [International Journal of Genomics](https://www.hindawi.com/journals/ijg/2017/8614160/) 2017.
10.	Peng Z, Liu F, Wang L, Zhou H, **Paudel DR**, Tan L, Maku J, Gallo M, Wang J. 2017a. Transcriptome profiles reveal gene regulation of peanut (*Arachis hypogaea* L.) nodulation. [Scientific Reports](https://www.nature.com/articles/srep40066) 7:40066.
11.	Peng Z, Fan W, Wang L, **Paudel DR**, Leventini D, Tillman BL, Wang J. 2017b. Target enrichment sequencing in cultivated peanut (*Arachis hypogaea* L.) using probes designed from transcript sequences. [Molecular Genetics and Genomics](https://link.springer.com/article/10.1007%2Fs00438-017-1327-z): 1–11.
12.	Adhikari L, Razar RM, **Paudel DR**, Ding R, Missaoui AM. 2017. Insights into seasonal dormancy of perennial herbaceous forages. [American Journal of Plant Sciences](https://m.scirp.org/papers/79625) 08: 2650–2680.
13.	**Paudel DR**, Dhakal P, Timsina KP, Dahal A. 2015. Azolla as an economic substitute to soybean based feed for poultry. [International Journal of Applied Sciences and Biotechnology](https://www.nepjol.info/index.php/IJASBT/article/view/13636/11507) 3: 619.
14.	Dhakal P, **Paudel DR**, Baral DR. 2015. Inventory of non-timber forest products in western Nepal and strategies for sustainable management. [International Journal of Environment](https://www.nepjol.info/index.php/IJE/article/view/15707/12642) 4: 130–139.
15.	**Paudel DR**, Hequet EF, Abidi N. 2013. Evaluation of cotton fiber maturity measurements. [Industrial Crops and Products](https://www.sciencedirect.com/science/article/abs/pii/S0926669013000095) 45: 435–441.

#### Book Chapters:

1. 	**Paudel D**, Dhakal S, Parajuli S, Adhikari L, Peng Z, You Q, Shahi D, Avci M, Makaju S, Kannan B. 2020. Use of quantitative trait loci to develop stress tolerance in plants. In [Plant Life under Changing Environment. Elsevier](https://www.sciencedirect.com/science/article/pii/B9780128182048000485).
2. 	 Peng Z, **Paudel D**, Wang L, Luo Z, You Q, Wang J. 2020 Methods for target enrichment sequencing via probe capture in legumes. In [Legume Genomics: Methods and Protocols](https://link.springer.com/protocol/10.1007%2F978-1-0716-0235-5_10) (Methods in Molecular Biology). Springer.

#### Extension Publications: 

1.	**Paudel D**. 2017. Rbiology: R tutorials for biologists. Available from: [rbiology.github.io](https://rbiology.github.io/)
2.	**Paudel D**. 2017. Informational interview with Dr. Marcio Resende for Conviron Scholars Program. [YouTube](https://www.youtube.com/watch?v=Ow5YgIi7FA4)
3.	**Paudel D**. 2010. Cultivation practice of Oyster Mushroom (Booklet - in Nepali). 2010. Youth Action Fund, Open Society Institute - Budapest.
4.	**Paudel D**. 2009. Urban farming for food security. Agri-Transformation. Year 1. Volume 1. Hitkari Samiti. Rampur Campus. Nepal.
5.	**Paudel D**. 2008. Azolla – a boon for farmers (In Nepali). Krishi Sandesh. Post Graduate Society, IAAS, Rampur, Nepal.

#### Conference and Workshop Talks:
1.	**Paudel D**, Wang L, Acharya J, de Souza C, Rios E, Wang J. 2020. Elucidating the effects of organic vs conventional cropping practice and rhizobia inoculation on peanut yield and rhizosphere microbial diversity. NAPA Conference 2020. Virtual.
2.	**Paudel D**, Dareus R, Rios E. 2020. Genome wide association study reveals candidate genes for flowering time in cowpea. ASA, CSSA and SSSA International Annual Meetings. Virtual.
3.	**Paudel D**, Parajuli S. 2020. Genomics and diversity of Nepalese rice genotypes. Agriculture and Forestry University. Virtual.
4.	**Paudel D**, Kannan B, Altpeter F, Wang J. 2019. Breeding, genetics, and genomics of napiergrass (elephant grass, *Cenchrus purpureus*). NSA-UF Talk Series, University of Florida.
5.	**Paudel D**, Kannan B, Altpeter F, Wang J. 2018. Utilizing cytoplasmic male sterility and interspecific hybridization to reduce invasiveness of elephant grass (*Cenchrus purpureus*). Agronomy Graduate Research Symposium, University of Florida.
6.	**Paudel D**, Kannan B, Altpeter F, Wang J. 2017a. First genetic linkage map of napiergrass (*Cenchrus purpureus*). Agronomy Graduate Research Symposium, University of Florida.
7.	**Paudel D**, Kannan B, Altpeter F, Wang J. 2017b. Constructing high density genetic map of napiergrass (*Cenchrus purpureus*). ASA-CSSA-SSSA Annual Meeting.
8.	**Paudel D**, Kelly B, Manandhar R, Hosseinali F, Abidi N, Hequet EF. 2012a. Evaluation of various measurements for assessing maturity of cotton fibers. Annual Biological Sciences Symposium, Texas Tech University.
9.	**Paudel D**, Kelly B, Manandhar R, Hosseinali F, Abidi N, Hequet EF. 2012b. Quantifying sample size requirements for AFIS Short Fiber Content measurements. Beltwide Cotton Conference.
10.	**Paudel D**, Timsina K. 2009. Utilizing azolla as an economical and sustainable feed to poultry. Global Future Institute Research Highlights, Rampur, Nepal.

#### Conference and Workshop Posters 

1.	**Paudel D**, Dareus R, Rios E. 2020. Genome wide association study reveals candidate genes for flowering time in cowpea. Plant Science Symposium, University of Florida.
2.	**Paudel D**, de Bem Oliveira I, Kenworthy KE, Unruh B, Meeks M, Milla-Lewis SR, Wu Y, Schwartz BM, Zhang J, Munoz PR, Rios E. 2019. Multi-environment evaluation of turfgrass species for turf quality in drought. ASA, CSSA and SSSA International Annual Meetings.
3.	**Paudel D**, Kannan B, Altpeter F, Wang J. 2018. Constructing a high-density genetic map of napiergrass (*Cenchrus purpureus* Schumach). Frontiers in Genetic Evaluation, Corteva Agriscience.
4.	Maya S\*, **Paudel D**, Wang J. 2017. Sequencing and annotating a *Bradyrhizobium* strain, Lb8. Undergraduate Research Symposium, University of Florida.
5.	**Paudel D**, Kannan B, Wang J, Altpeter F. 2016. Developing marker assisted breeding resources in napiergrass and interspecific hybridization with pearl millet for improved biomass production and biosafety. National Association of Plant Breeders Annual Meeting.
6.	**Paudel D**, Kannan B, Wang J, Altpeter F. 2015. Interspecific hybridization between pearl millet and napiergrass for biosafety and identification of QTLs for flowering time in napiergrass. Florida Genetics Symposium.
7.	Hosseinali F, Kelly B, Manandhar R, **Paudel D**, Abidi N, Hequet EF. 2012. Variability of individual cotton fiber tensile properties within and between samples. Beltwide Cotton Conference.
8.	Kelly B, Manandhar R, Hosseinali F, **Paudel D**, Abidi N, Hequet EF. 2012. Decoding distributional changes in fiber quality during consecutive stages of processing. Beltwide Cotton Conference.
9.	Manandhar R, Kelly B, Hosseinali F, **Paudel D**, Abidi N, Hequet EF. 2012. Relationship between individual fiber length and linear density within-sample. Beltwide Cotton Conference.
10.	**Paudel D**, Kelly B, Manandhar R, Hosseinali F, Abidi N, Hequet EF. 2012. Assessing rapid measurements of the secondary cell wall development of cotton fibers. Second Annual Agricultural Consortium of Texas Research Symposium.

(\*undergraduate student under my supervision)
